system-tests/project-fixtures/react/src/App.cy.jsx

```diff
@@ -4,5 +4,5 @@ import { App } from './App'
 
 it('renders hello world', () => {
   mount(<App />)
-  cy.get('h1').contains('Hello World')
+  cy.get('h1').contains('Hello World').click()
```

Comment:
Do we need this `click()`?

Reply:
Yes. Without it, if we are displaying an overlay the tests will pass (even though they shouldn't). I can maybe throw a comment on there to make that more clear why I'm doing it.

Reply:
Comment added here: [`fd175c8` (#21599)](https://github.com/cypress-io/cypress/pull/21599/commits/fd175c863f92b811319ae4792a1ee940cc8a3e2f)

Reply:
`.should('be.visible')` btw. Though `.click()`'s actionability checks are more aggressive (and will cover more)